layouts/partials/help.html

```diff
@@ -3,25 +3,33 @@
     <div class="container">
         <div class="row">
             <div class="col-lg-12 text-center">
-                <h2 class="section-heading">{{ with .Site.Params.help.title }}{{ . }}{{ end }}</h2>
-                <h3 class="section-subheading text-muted">{{ with .Site.Params.help.subtitle }}{{ . | markdownify }}{{ end }}</h3>
```

Comment:
I didn't see it in that part, I don't think... Do you want me to throw it in there somewhere?